README.md

```diff
@@ -36,12 +36,17 @@ build one with its internal builder:
 ```java
 Request request = Request.builder()
         .method("POST")
-        .uriWithQuery(URI.create("https://localhost/service/v2/users"))
-        .body("{ field: \"foo\" }")
-        .header("Content-Type", "application/json")
+        .uriWithQuery(URI.create("https://akaa-baseurl-xxxxxxxxxxx-xxxxxxxxxxxxx.luna.akamaiapis.net/billing-usage/v1/reportSources"))
```

Comment:
I took care of this in a very elaborate fashion, and I think it invalidates part of pull request 12 as a result. I changed the method name and field name to "uri" and left a deprecated wrapper method by the old name. I also added a variant which takes a string and called `URI#create()` on it.

The result here is that `RequestBuilder` will take an absolute or relative use and just pluck out the bits it cares about. The rest will be discarded. The Javadoc indicates this.